### ISSUE TITLE
Nemo freezes on "Preparing to move" window

### ISSUE THREAD

**Issue by fedy2** [linuxmint/nemo#877]
I'm trying to move a bunch of files from a folder to another but Nemo freezes on "Preparing to move" window. I have to kill it because it can't be used anymore. 
Tried to move those files again but I'm getting the same behavior.

**Comment by OldGrampa** [comment]
I'm having similar problem.
In Ubuntu 14.10, Nemo worked fine til about 2 weeks ago,
now many times a 1-2 minute pause before file moves.

**Comment by sevenskull** [comment]
I am having this same issue in Mint 17.2 when i move more a lot of files (sometimes 200 freeze).
There is another thread, that maybe is related: https://github.com/linuxmint/nemo/issues/933

**Comment by alhirzel** [comment]
Also happening for me on a selection of 278 files. I was moving from a directory to a subfolder on an SSD.

Linux SNAKE 4.2.5-1-ARCH #1 SMP PREEMPT Tue Oct 27 08:13:28 CET 2015 x86_64 GNU/Linux

nemo 2.8.5

**Comment by mxmlnkn** [comment]
I'm also having this problem with nemo 2.8.7 (and prior versions) on debian even when moving only ~100 files. Often it doesn't completely freeze, I just have to wait 5 minutes.

**Comment by mxmlnkn** [comment]
I'm still battling with this bug daily. File operations with nemo are basically non-working for me. I often have to use the command line which somewhat defeats the purpose of using nemo in the first place. I used `wmctrl -lp` to find the pid of the hanging nemo file operation and then used `strace -s 200 -ffp 6449 2> nemo-freeze-on-99percent-strace.log` to maybe help some developer to see where nemo hangs. The output grows large very fast, so I hat to edit it a bit:

    strace: Process 6449 attached with 4 threads
    [pid  6451] restart_syscall(<... resuming interrupted restart_syscall ...> <unfinished ...>
    [pid  6454] restart_syscall(<... resuming interrupted restart_syscall ...> <unfinished ...>
    [pid  6450] restart_syscall(<... resuming interrupted poll ...> <unfinished ...>
    [pid  6449] restart_syscall(<... resuming interrupted poll ...>) = 1
    [pid  6449] read(3, 0x7ffdd2817c40, 16) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, {msg_name(0)=NULL, msg_iov(1)=[{"l\4\1\1\25\0\0\0\203\244o\0\205\0\0\0\1\1o\0\30\0\0\0/org/a11y/atspi/registry\0\0\0\0\0\0\0\0\2\1s\0\27\0\0\0org.a11y.atspi.Registry\0\3\1s\0\31\0\0\0EventListenerDeregistered\0\0\0\0\0\0\0\10\1g\0\2ss\0\7\1s\0\4\0\0\0:1.4\0\0\0\0\t\0\0\0:1.995425\0\0\0\0\0\0\0\0", 2048}], msg_controllen=0, msg_flags=MSG_CMSG_CLOEXEC}, MSG_CMSG_CLOEXEC) = 173
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, 0x7ffdd2817ab0, MSG_CMSG_CLOEXEC) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] read(3, "\3\0\0\0\0\0\0\0", 16) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=5, revents=POLLIN}])
    [pid  6449] read(3, 0x7ffdd2817c40, 16) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, {msg_name(0)=NULL, msg_iov(1)=[{"l\4\1\1\25\0\0\0\212\244o\0\205\0\0\0\1\1o\0\30\0\0\0/org/a11y/atspi/registry\0\0\0\0\0\0\0\0\2\1s\0\27\0\0\0org.a11y.atspi.Registry\0\3\1s\0\31\0\0\0EventListenerDeregistered\0\0\0\0\0\0\0\10\1g\0\2ss\0\7\1s\0\4\0\0\0:1.4\0\0\0\0\t\0\0\0:1.995426\0\0\0\0\0\0\0\0", 2048}], msg_controllen=0, msg_flags=MSG_CMSG_CLOEXEC}, MSG_CMSG_CLOEXEC) = 173
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, 0x7ffdd2817ab0, MSG_CMSG_CLOEXEC) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] read(3, "\3\0\0\0\0\0\0\0", 16) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1 <unfinished ...>
    [pid  6450] <... restart_syscall resumed> ) = 0
    [pid  6450] inotify_add_watch(11, "", IN_MODIFY|IN_ATTRIB|IN_CLOSE_WRITE|IN_MOVED_FROM|IN_MOVED_TO|IN_CREATE|IN_DELETE|IN_DELETE_SELF|IN_MOVE_SELF|IN_UNMOUNT|IN_ONLYDIR) = -1 ENOENT (No such file or directory)
    [pid  6450] poll([{fd=8, events=POLLIN}, {fd=11, events=POLLIN}], 2, 3999 <unfinished ...>
    [pid  6449] <... poll resumed> )        = 1 ([{fd=5, revents=POLLIN}])
    [pid  6449] read(3, 0x7ffdd2817c40, 16) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, {msg_name(0)=NULL, msg_iov(1)=[{"l\4\1\1\25\0\0\0\221\244o\0\205\0\0\0\1\1o\0\30\0\0\0/org/a11y/atspi/registry\0\0\0\0\0\0\0\0\2\1s\0\27\0\0\0org.a11y.atspi.Registry\0\3\1s\0\31\0\0\0EventListenerDeregistered\0\0\0\0\0\0\0\10\1g\0\2ss\0\7\1s\0\4\0\0\0:1.4\0\0\0\0\t\0\0\0:1.995427\0\0\0\0\0\0\0\0", 2048}], msg_controllen=0, msg_flags=MSG_CMSG_CLOEXEC}, MSG_CMSG_CLOEXEC) = 173
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, 0x7ffdd2817ab0, MSG_CMSG_CLOEXEC) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] read(3, "\3\0\0\0\0\0\0\0", 16) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=5, revents=POLLIN}])
    [pid  6449] read(3, 0x7ffdd2817c40, 16) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, {msg_name(0)=NULL, msg_iov(1)=[{"l\4\1\1\25\0\0\0\230\244o\0\205\0\0\0\1\1o\0\30\0\0\0/org/a11y/atspi/registry\0\0\0\0\0\0\0\0\2\1s\0\27\0\0\0org.a11y.atspi.Registry\0\3\1s\0\31\0\0\0EventListenerDeregistered\0\0\0\0\0\0\0\10\1g\0\2ss\0\7\1s\0\4\0\0\0:1.4\0\0\0\0\t\0\0\0:1.995428\0\0\0\0\0\0\0\0", 2048}], msg_controllen=0, msg_flags=MSG_CMSG_CLOEXEC}, MSG_CMSG_CLOEXEC) = 173
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, 0x7ffdd2817ab0, MSG_CMSG_CLOEXEC) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] read(3, "\3\0\0\0\0\0\0\0", 16) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=5, revents=POLLIN}])
    [pid  6449] read(3, 0x7ffdd2817c40, 16) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, {msg_name(0)=NULL, msg_iov(1)=[{"l\4\1\1\25\0\0\0\237\244o\0\205\0\0\0\1\1o\0\30\0\0\0/org/a11y/atspi/registry\0\0\0\0\0\0\0\0\2\1s\0\27\0\0\0org.a11y.atspi.Registry\0\3\1s\0\31\0\0\0EventListenerDeregistered\0\0\0\0\0\0\0\10\1g\0\2ss\0\7\1s\0\4\0\0\0:1.4\0\0\0\0\t\0\0\0:1.995429\0\0\0\0\0\0\0\0", 2048}], msg_controllen=0, msg_flags=MSG_CMSG_CLOEXEC}, MSG_CMSG_CLOEXEC) = 173
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, 0x7ffdd2817ab0, MSG_CMSG_CLOEXEC) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] read(3, "\3\0\0\0\0\0\0\0", 16) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=4, revents=POLLIN}])
    [pid  6449] read(3, 0x7ffdd2817c40, 16) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] recvmsg(4, {msg_name(0)=NULL, msg_iov(1)=[{"#\203QA\v\0\0\0\7\0\2\0m\371z1\17\0\0\3\244\2\0\0\37\345\205\1\0\0\0\0\0\0\367\7\0\0,\0\0\0\30\0\0\0\2\0\1\0\1\0\0\0\0\0\0\0\0\0\20\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0", 4096}], msg_controllen=0, msg_flags=0}, 0) = 76
    [pid  6449] recvmsg(4, 0x7ffdd2817ab0, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 0 (Timeout)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 0 (Timeout)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 0 (Timeout)
    [pid  6449] poll([{fd=4, events=POLLIN|POLLOUT}], 1, -1) = 1 ([{fd=4, revents=POLLOUT}])
    [pid  6449] writev(4, [{"\203*\4\0\37\345\205\1\0\0\0\0\2\0\210\1", 16}, {NULL, 0}, {"", 0}], 3) = 16
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 500) = 1 ([{fd=4, revents=POLLIN}])
    [pid  6449] recvmsg(4, {msg_name(0)=NULL, msg_iov(1)=[{"#\203QA\32\0\0\0\6\0\2\0m\371z1\0\0\0\0\244\2\0\0\37\345\205\1\0\0\0\0\200\373\367\7!q,\0\200\373\30\0!q\2\0\10\0\2\0\17\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\20\0\0\0\20\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\3\0\0\0\0\0\0\0\367\7\0\0\345w~\373,\0\0\0G\253!q", 4096}], msg_controllen=0, msg_flags=0}, 0) = 136
    [pid  6449] recvmsg(4, 0x7ffdd2817ab0, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] recvmsg(4, 0x7ffdd2817960, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 0 (Timeout)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 0 (Timeout)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 0 (Timeout)
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 500) = 1 ([{fd=4, revents=POLLIN}])
    [pid  6449] recvmsg(4, {msg_name(0)=NULL, msg_iov(1)=[{"#\203RA\v\0\0\0\10\0\2\0q\371z1\17\0\0\3\244\2\0\0\37\345\205\1\0\0\0\0\0\0a\10\0\0\4\0\0\0\202\0\0\0\332\377\1\0\1\0\0\0\0\0\0\0\0\0\20\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0", 4096}], msg_controllen=0, msg_flags=0}, 0) = 76
    [pid  6449] recvmsg(4, 0x7ffdd2817ab0, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 0 (Timeout)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 0 (Timeout)
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=5, revents=POLLIN}])
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, {msg_name(0)=NULL, msg_iov(1)=[{"l\4\1\1\25\0\0\0\254\244o\0\205\0\0\0\1\1o\0\30\0\0\0/org/a11y/atspi/registry\0\0\0\0\0\0\0\0\2\1s\0\27\0\0\0org.a11y.atspi.Registry\0\3\1s\0\31\0\0\0EventListenerDeregistered\0\0\0\0\0\0\0\10\1g\0\2ss\0\7\1s\0\4\0\0\0:1.4\0\0\0\0\t\0\0\0:1.995430\0\0\0\0\0\0\0\0", 2048}], msg_controllen=0, msg_flags=MSG_CMSG_CLOEXEC}, MSG_CMSG_CLOEXEC) = 173
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, 0x7ffdd2817ab0, MSG_CMSG_CLOEXEC) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, 0) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=3, revents=POLLIN}])
    [pid  6449] read(3, "\3\0\0\0\0\0\0\0", 16) = 8
    [pid  6449] recvmsg(4, 0x7ffdd2817a90, 0) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}, {fd=5, events=POLLIN}], 3, -1) = 1 ([{fd=5, revents=POLLIN}])
    [pid  6449] read(3, 0x7ffdd2817c40, 16) = -1 EAGAIN (Resource temporarily unavailable)
    [pid  6449] write(3, "\1\0\0\0\0\0\0\0", 8) = 8
    [pid  6449] recvmsg(5, {msg_name(0)=NULL, msg_iov(1)=[{"l\4\1\1\25\0\0\0\260\244o\0\205\0\0\0\1\1o\0\30\0\0\0/org/a11y/atspi/registry\0\0\0\0\0\0\0\0\2\1s\0\27\0\0\0org.a11y.atspi.Registry\0\3\1s\0\31\0\0\0EventListenerDeregistered\0\0\0\0\0\0\0\10\1g\0\2ss\0\7\1s\0\4\0\0\0:1.4\0\0\0\0\t\0\0\0:1.995431\0\0\0\0\0\0\0\0", 2048}], msg_controllen=0, msg_flags=MSG_CMSG_CLOEXEC}, MSG_CMSG_CLOEXEC) = 173

If this doesn't help, maybe someone can show me what to do/debug when this occurs again.

**Comment by icarter09** [comment]
@mxmlnkn are you still experiencing this issue on the latest version of nemo?

**Comment by mxmlnkn** [comment]
Well, I'm using Caja these days, but I reinstalled Nemo eager to test this.

I can't seem to reproduce the bug with Nemo 4.0.6, I could even copy 3k files to another storage device. A file move progress window would have been nice though. At least for my tests nothing did open.

Other problems while trying to reproduce the bug:
 - In some folders with a bit more files, I can't switch from the default Icon View to the List View nor the Compact View. The buttons can be pressed but they won't keep their "pressed" state and nothing changes. The console fills up for a while with warnings like "Error creating thumbnail for ...jpg", looking at those files, they have the wrong extensions and are instead Web/P images. Maybe that could also be supported? After going to another folder and switching back to the folder in which I changed the view, the newly selected view actually seems to be shown.
 - For Caja, I can set custom folder icons using `gio set . metadata::custom-icon file.ico`, however, nemo, still does seem to not support these.
 - The preferences window seems to ignore "escape" and unexpectedly does not close on pressing it. For other windows, like "Properties", it works fine, though. It doesn't even have a Cancel button. I have to click the close button in the top right, which is something I almost never do, as at least Ctrl+W or escape work everywhere else.
 - Setting the default zoom level in Edit-Preferences only seems to have been applied the very first time on the folder open in the background. After that every further change does not do anything anymore and only after changing to a completely fresh / unseen folder, this works another one time again. Furthermore, the "default" zoom seems to now be the custom fixed zoom for those folders I had open while changing that. I.e., after changing the Edit -> Preferences -> default zoom while having open folder A to 50% and then while having open folder B to 400%, those zooms are fixed and folder A will still be 50% even though the default is now 400%. I'm not able to change that back except for Ctrl+Wheel.
